### ISSUE TITLE
ebook creation in simple API documentation fails

### ISSUE THREAD

**Issue by bgrusdt** [freme-project/freme-project.github.io#222]
the in the 'Simple API Documentation' , the call 
POST /e-publishing/html  complains about the input format when the provided json metadata is used for the metadata parameter together with the alice.zip data. 

```
 "exception": "eu.freme.common.exception.BadRequestException",
  "path": "/e-publishing/html",
  "message": "Unsupported informat='multipart/form-data'.
```

**Comment by jnehring** [comment]
I can reproduce the bug. @ArneBinder please find out what is wrong here.

**Comment by ArneBinder** [comment]
Should be fixed, the reason of the bug is described in #239.
